src/libsystemd/sd-login/sd-login.c

```diff
@@ -134,6 +135,206 @@ _public_ int sd_pid_get_cgroup(pid_t pid, char **cgroup) {
         return 0;
 }
 
+_public_ int sd_pidfd_get_session(int pidfd, char **session) {
```

Comment:
rename `session` to `ret` or `ret_session` matching our current coding style?

Reply:
I wanted to keep the signatures matching the old methods, if they are different it might wrongly suggest the return value is different

Reply:
i'll change it in the implementation, but leave the public header alone, they don't have to match after all